cmd/release-controller/http.go

```diff
@@ -708,7 +708,7 @@ func (c *Controller) httpReleaseInfo(w http.ResponseWriter, req *http.Request) {
 
 	renderInstallInstructions(w, mirror, info.Tag, tagPull, c.artifactsHost)
 
-	renderVerifyLinks(w, *info.Tag, info.Release)
+	renderVerifyLinks(w, info.Tag, info.Release)
```

Comment:
Are you 100% sure nothing in renderVerifyLinks fiddles with the tag value?

Reply:
Also, where's the signature change for `renderVerifyLinks`?  In a different commit?

Reply:
reverted the signature change in the next [commit](https://github.com/openshift/release-controller/pull/137/commits/a3a9d6c20f41fa84ae760e1f8845b039ed022a3c).

renderVerifyLinks shouldn't modify the tags, it just reads the phase and verify annotation.